details.md

```diff
@@ -33,9 +33,9 @@ We are also looking to secure **sponsorship** which will go directly towards bri
 ### Accommodation
 
 There are twin and single rooms, all with en-suite facilities. The cost, without sponsorship discounts, will be:
-<p>1. Thu 7 June to Sun 10 June (3 nights) – Single room: £250.00 non-refundable deposit + £207.50 (paid on checkout)</p>
-<p>2. Thu 7 June to Sun 10 June (3 nights) – Twin room: £250.00 non-refundable deposit + £177.5 (paid on checkout)</p>
-(Please note that Eventbrite fees apply)
+<p>1. Thu 7 June to Sun 10 June (3 nights) – Single room: Price TBD</p>
+<p>2. Thu 7 June to Sun 10 June (3 nights) – Twin room: Price TBD</p>
```

Comment:
@tomphp should we remove the dates here or use the provisional dates?

Reply:
I'll just remove the information completely until we have the date.